src/components/GlobalHelper/GlobalHelper.st.css

```diff
@@ -0,0 +1,90 @@
+:import {
+  -st-from: "wix-ui-core/dist/src/baseComponents/Popover/Popover.st.css";
+  -st-default: Popover;
+}
+
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: brandSecondary, backgroundSecondary
+}
+
+:import {
+  -st-from: "../../typography.st.css";
+  -st-named: fontRoman;
+}
+
+.root {
+  -st-states: bounce, placement-right, placement-left, placement-top, placement-bottom;
+  /* -st-extends: Tooltip; */
+  -st-extends: Popover;
+  white-space: nowrap;
+}
+
+/* popoverContent and innerContent are taken from Tooltip.tooltipContent,
+ * but I split them.
+*/
+.root::popoverContent {
+  border-radius: 8px;
+  padding: 12px 24px;
+  box-sizing: border-box;
+  box-shadow:
+  0 6px 6px 0 rgba(22, 45, 60, 0.06),
+  0 0 18px 0 rgba(22, 45, 60, 0.12);
+  
+  border: none;
+  opacity: 0.95;
+  
+  /* max-width: 230px; */
+  background: value(brandSecondary);
+}
+
+.innerContent {
+  font-family: value(fontRoman);
+  text-align: center;
+  color: value(backgroundSecondary);
+  word-wrap: break-word;
+  -webkit-font-smoothing: antialiased;
+  font-size: 14px;
+  line-height: 1.29;
+}
+
+.root:bounce::popover {
+  animation-duration: 1.15s;
+  animation-iteration-count: infinite;
+}
+
+.root:bounce:placement-right::popover {
```

Comment:
are you sure you need the bouncing capability?

Reply:
I asked. ben said yes.